src/app/pages/common/entity/table/table.component.ts

```diff
@@ -31,8 +31,11 @@ export interface InputTableConf {
   tableComponent?: TableComponent;
   emptyEntityLarge?: boolean;
   parent: any;
+  expandable?: boolean;
```

Comment:
Please make a separate interface for ExpandableTable.

Reply:
Separate interface was made for ExpandableTable. Here, the original table needs to know if itself is an expandable table so it can decide not to show it's header.

Reply:
Okay, can you at least add a new line between old and new fields and add a comment, so that we know in the future that this field is not "real"?